openpype/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -25,27 +25,48 @@ class SubstanceLoadProjectMesh(load.LoaderPlugin):
     icon = "code-fork"
     color = "orange"
 
-    options = [
-        qargparse.Boolean(
-            "preserve_strokes",
-            default=True,
-            help="Preserve strokes positions on mesh.\n"
-                 "(only relevant when loading into existing project)"
-        ),
-        qargparse.Boolean(
-            "import_cameras",
-            default=True,
-            help="Import cameras from the mesh file."
-        )
-    ]
-
-    def load(self, context, name, namespace, data):
+    @classmethod
+    def get_options(cls, contexts):
+        project_workflow_option = {
```

Comment:
I feel like this is 'plural'?
```suggestion
        project_workflow_options = {
```

Plus you're using this for the setting `project_uv_workflow` on the attribute definitions for the argument `items`.

As such, I would actually name this:
```suggestion
        project_uv_workflow_items = {
```